README.md

```diff
@@ -24,7 +24,7 @@ Install the loader in your project:
 npm install --save-dev @funboxteam/scss-vars-loader
 ```
 
-Add it into the project's Webpack config so that it is called before sass-loader:
```

Comment:
But it was correct.

> Loaders are evaluated/executed from right to left (or from bottom to top)

https://webpack.js.org/concepts/loaders/#configuration

scss-vars-loader should be called _before_ the sass-loader, because scss-vars-loader adds SCSS code that must be processed by sass-loader.

Reply:
Oh, didn't know about it, thanks!